pkgs/top-level/all-packages.nix

```diff
@@ -26824,23 +26824,40 @@ with pkgs;
   dm-sans = callPackage ../data/fonts/dm-sans { };
   dns-root-data = callPackage ../data/misc/dns-root-data { };
 
-  docbook5 = callPackage ../data/sgml+xml/schemas/docbook-5.0 { };
+  inherit
+    (lib.attrsets.mapAttrs'
+     (relVer: deriv: lib.attrsets.nameValuePair
+      ("docbook_" + lib.strings.concatStringsSep "_" (lib.strings.splitString "." relVer))
+      deriv)
+     (callPackage ../by-name/do/docbook/versions.nix {}))
```

Comment:
Now that I am looking for this, it looks less elegant than what I thought initially.
I am not understanding this code, and why the CI accepted this.

Reply:
I just wanted to repeat less, but it seems to make no sense, and much complicate the code.

Reply:
Because it does not conflict with the by-name RFC. But this is definitely not the intended use of by-name, imo